### ISSUE TITLE
1.0+WSI_Extensions spec contains all ratified extensions

### ISSUE THREAD

**Issue by HildarTheDorf** [KhronosGroup/Vulkan-Docs#2456]
The 1.0+WSI spec (e.g. available here https://registry.khronos.org/vulkan/specs/1.0-wsi_extensions/html/vkspec.html) actually contains all extensions.

I have previously suggested this spec to beginners with vulkan as it's easier to read than the 1.3+all_extensions spec, however that advantage is lessened somewhat with so many extensions in the text.

Can this build be restored to being just 1.0+WSI? Or if not, can the website be corrected to not contain a broken link?

**Comment by oddhack** [comment]
TL;DR we're probably not going to do either of those things. The 'wsi_extensions' part of the URL is legacy and just there to avoid breaking external links. As more KHR extensions were added we started using 'khr_extensions' in URLs instead and, now that EXT extensions are being ratified as well, that terminology is also a bit dated. The descriptions in the registry index page are accurate and the specs themselves describe what they contain, so I'm not eager to spend time modifying Apache configuration files and build scripts just to tweak the URLs.

Our intent is to just publish the core + all extensions spec at some point, especially as the end state will be for most EXT extensions to be ratified. All the other spec variants would be redirected at that point. The problem is that it's quite burdensome to publish all the many spec artifacts we do today, yet still impossible to satisfy everyone's preferences. People are  welcome to build their own spec artifacts with whatever combination of extensions and core versions they want, and we intend to keep supporting that functionality.

Eventually we might do a variant of the specification that just documents modern functionality and omits legacy extensions and APIs where newer ways are available - but that is not imminent (or promised).

**Comment by oddhack** [comment]
Closing per discussion in the teleconference this morning.
